index.html

```diff
@@ -1083,6 +1090,84 @@ <h3>Multikey</h3>
 
           </section>
 
+          <section>
+            <h3>JsonWebKey</h3>
+            <p>
+The JSON Web Key (JWK) data model is a specific type of <a>verification method</a>
+that uses the JWK specification [[RFC7517]] to encode key types into a
+set of parameters. To encode a JSON Web Key, the <a>verification method</a>
+`type` MUST be set to `JsonWebKey` and the `publicKeyJwk` or `secretKeyJwk`
+value MUST be a valid JWK value as described in [[RFC7517]]. An example of a
+JSON Web Key is provided below:
+            </p>
+
+            <pre class="example nohighlight"
+              title="JSON Web Key encoding of an Ed25519 public key">
+{
+  "@context": ["https://www.w3.org/ns/security/jwk/v1"],
+  "id": "did:example:123456789abcdefghi#key-1",
+  "type": "JsonWebKey",
+  "controller": "did:example:123456789abcdefghi",
+  "publicKeyJwk": {
+    "kty": "OKP",
+    "alg": "EdDSA"
+    "crv": "Ed25519",
+    "kid": "key-1",
+    "x": "_1EiHquO2aUx9JARSu0P8jdYT_OVneYxYOnOMAmUcFI",
+  }
+}
```

Comment:
Why do we need yet another context here? The definition of `publicKeyJwk` stipulates that its object is of datatype JSON, meaning that the properties within that object are ignored by any JSON-LD processing. If so, then I do not see any reason using a different context (which I have not seen defined anywhere).

Proposal: just use the usual security context here.

Reply:
We don't need another context here.

I will object if one is required.

Reply:
Also, seems we are following a different style here than the core context, where we seem to prefer bundling, that inconsistency, feels... not good to me.

Reply:
Raised https://github.com/w3c/vc-data-integrity/issues/164 to track @OR13's concern, which seems to be more of a misunderstanding than a misalignment.